### ISSUE TITLE
Add ability to see esl flagged esp for SkyrimVR for merging

### ISSUE THREAD

**Issue by DonFlymoor** [Nexus-Mods/Vortex#12579]
Is your feature request related to a problem? Please describe.
I quickly run into the 255 plugin limit. Using zMerge, I can merge all esl flagged esp's into one esp plugin with no observable gameplay issues, saving a lot of space for more mods.

All esl plugins can be safely merged, therefore I would like to know which ones are flagged esl so as to manually merge them, thus mimicking the behavior of esl flagged plugins in Skyrim, all esl flagged esps taking one esp slot

Describe the solution you'd like
Make plugins sortable by the esl tag  so I know which ones I should merge. Not "Light" and "Possibly Light" as is done for SkyrimSE, but "esl flagged esp" and "possible to be esl flagged'

**Comment by Pickysaurus** [comment]
Duplicate of this: https://github.com/Nexus-Mods/Vortex/issues/12394

**Comment by DonFlymoor** [comment]
Indeed, BUT that issue was closed without actually addressing my problem.

**Comment by DonFlymoor** [comment]
You said "Light plugins are not supported by Skyrim VR so it would be confusing to show that information." - that is true
but my statement is:
All esl plugins can be safely merged, therefore I would like to know which ones are flagged esl so as to manually merge them, thus mimicking the behavior of esl flagged plugins in Skyrim, all esl flagged esps taking one esp slot

**Comment by DonFlymoor** [comment]
In the skyrimVR mods, plugins flagged as esp cannot be filtered by plugins flagged as light.
